### PR TITLE
DAT-18148: refactorings + tests

### PR DESCRIPTION
**Pull request by filipelautert**
The following changeset will create auto increment/generated columns:

```
    <changeSet author="testuser" id="4">
        <createTable tableName="tableac" >
            <column name="longcolumn" type="long" autoIncrement="true" generationType="IDENTITY" />
            <column name="eventTime" type="timestamp"/>
            <column name="year" type="int" defaultValueComputed="GENERATED ALWAYS AS (YEAR(eventTime))"/>
            <column name="eventDate" type="date" defaultValueComputed="GENERATED ALWAYS AS (CAST(eventTime AS DATE))" />
            <column name="eventDescription" type="string">
                <constraints nullable="false" />
            </column>
            <column name="eventShortDescription" type="string" defaultValueComputed="GENERATED ALWAYS AS (SUBSTRING(eventDescription, 0, 1))" />
        </createTable>
    </changeSet>
    <changeSet id="5" author="filipe">
        <insert tableName="tableac">
            <column name="eventTime" value="2022-01-01 00:00:00"/>
            <column name="eventDescription" value="My first event"/>
        </insert>
    </changeSet>
```

Nothing was required on databricks side, please see companion PR in core: https://github.com/liquibase/liquibase/pull/6263

### CODE REVIEW

File: src/main/java/liquibase/ext/databricks/database/DatabricksDatabase.java
Comment (filipelautert) [inline src/main/java/liquibase/ext/databricks/database/DatabricksDatabase.java:1]: No changes, just refactoring to make sonar happy.

File: src/test/resources/liquibase/harness/change/changelogs/databricks/createTableWithDefaultValues.xml
Comment (vitaliimak) [inline src/test/resources/liquibase/harness/change/changelogs/databricks/createTableWithDefaultValues.xml:1]: The `ChangeObjectTests` require to have the `expectedSnapshot` file to work.

Comment (vitaliimak) [reply src/test/resources/liquibase/harness/change/changelogs/databricks/createTableWithDefaultValues.xml:1]: Although the checkboxes are green, there are failed tests https://github.com/liquibase/liquibase-databricks/pull/175/checks?check_run_id=29317521268

Does the workflow have errors?

Comment (filipelautert) [reply src/test/resources/liquibase/harness/change/changelogs/databricks/createTableWithDefaultValues.xml:1]: About expectedSnapshots: I used createTableDataTypeText.xml as an example and there is no expectedSnapshot to it, just sql... weird

**Review by vitaliimak — APPROVED** [review]

**Comment by vitaliimak** [comment]
@filipelautert 
If you add an empty expectedSnapshot JSON file then the test will pass after the core change.
image

**Pull request merged**